cmd/krel/cmd/release_notes.go

```diff
@@ -239,67 +365,204 @@ func createDraftPR(tag string, result *releaseNotesResult) error {
+		return errors.Wrap(err, "writing assets.ts file")
+	}
+
+	return nil
+}
 
-	// commit the results
-	err = sigReleaseRepo.Add(filepath.Join("releases", fmt.Sprintf("release-%d.%d", s.Major, s.Minor), draftFilename))
+// processJSONOutput Runs NPM prettier inside repoPath to format the JSON output
+func processJSONOutput(repoPath string) error {
+	npmpath, err := exec.LookPath("npm")
 	if err != nil {
-		return errors.Wrap(err, "adding release notes draft to staging area")
+		return errors.Wrap(err, "while looking for npm in your path")
+	}
+
+	// run npm install
+	logrus.Info("Installing npm modules, this can take a while")
+	err = command.NewWithWorkDir(repoPath, npmpath, "install").RunSuccess()
+	if err != nil {
+		return errors.Wrap(err, "running npm install in kubernetes-sigs/release-notes")
 	}
 
-	err = sigReleaseRepo.Commit("Release Notes draft for k/k " + tag)
+	// run npm prettier
+	logrus.Info("Running npm prettier...")
+	err = command.NewWithWorkDir(repoPath, npmpath, "run", "prettier").RunSuccess()
 	if err != nil {
+		return errors.Wrap(err, "running npm prettier in kubernetes-sigs/release-notes")
+	}
+
+	return nil
+}
+
+// createWebsitePR creates the JSON version of the release notes and pushes them to a user fork
+func createWebsitePR(tag string, result *releaseNotesResult) error {
+	_, err := util.TagStringToSemver(tag)
+	if err != nil {
+		return errors.Wrapf(err, "no valid tag: %v", tag)
+	}
+
+	jsonNotesFilename := fmt.Sprintf("release-notes-%s.json", tag[1:])
+	branchname := "release-notes-json-" + tag
+
+	///
+	// checkout kubernetes-sigs/release-notes
+	k8sSigsRepo, err := prepareFork(
+		branchname, releaseNotesOpts.kubernetessigsForkPath, defaultKubernetesSigsOrg,
+		defaultKubernetesSigsRepo, releaseNotesOpts.websiteOrg, releaseNotesOpts.websiteRepo,
+	)
+	if err != nil {
+		return errors.Wrap(err, "preparing local fork branch")
+	}
+
+	// add a reference to the new json file in assets.ts
+	if err := addReferenceToAssetsFile(k8sSigsRepo.Dir(), jsonNotesFilename); err != nil {
+		return errors.Wrapf(err, "adding %s to assets file", jsonNotesFilename)
+	}
+
+	// generate the notes
+	jsonNotesPath := filepath.Join("src", "assets", jsonNotesFilename)
+	logrus.Debugf("Release notes json file will be written to %s", filepath.Join(k8sSigsRepo.Dir(), jsonNotesPath))
+	err = ioutil.WriteFile(filepath.Join(k8sSigsRepo.Dir(), jsonNotesPath), []byte(result.json), 0644)
+	if err != nil {
+		return errors.Wrapf(err, "writing release notes json file")
+	}
+
+	// Run NPM prettier
+	if err := processJSONOutput(k8sSigsRepo.Dir()); err != nil {
+		return errors.Wrap(err, "while formatting release notes JSON files")
+	}
+
+	// add the modified files & commit the results
+	if err := k8sSigsRepo.Add(jsonNotesPath); err != nil {
+		return errors.Wrap(err, "adding release notes draft to staging area")
+	}
+
+	if err := k8sSigsRepo.Add(filepath.Join("src", "environments", "assets.ts")); err != nil {
```

Comment:
As already mentioned, `filepath.Join("src", "environments", "assets.ts")` should be variable in my point of view.